templates/registration/login.html

```diff
@@ -0,0 +1,18 @@
+{% extends "base.html" %}
+
+
+{% block content %}
+  <h1>Login</h1>
+
+
+  {% if form.errors %}
+    <p style="color: red">Invalid credentials</p>
+  {% endif %}
+
+
+  <form method="post" action="{% url 'login' %}">
```

Comment:
The form action URL should be `{% url 'login' %}` only if you have explicitly defined a URL pattern named 'login'. If you are using Django's built-in authentication views, the correct URL name is typically 'login' without any namespace. Ensure that the URL name matches the one defined in your URL configuration.